favor.lic

```diff
@@ -159,7 +159,10 @@ class CrossingFavor
 
   def leave_room
     pause 1
-    bput('go stair', 'You find yourself more than eager to step into the inviting world that awaits you')
+    case bput('go stair', 'You find yourself more than eager to step into the inviting world that awaits you', 'braziers on either side suddenly flare greatly')
+    when 'braziers on either side suddenly flare greatly'
+      start_puzzle('tinders')
```

Comment:
❓ Should this `return`? Or perhaps put `start_puzzle('door')` in a separate `when`?

Reply:
it should automatically do start_puzzle('door') after the when is over to complete it? I'm just assuming it can only happen that you need to do the tinders puzzle twice.

Reply:
What I mean is, in this new case `start_puzzle('tinders')` will eventually call `leave_room` _again_, which will eventually result in `start_puzzle('door')`, and that function will return, then control will return to here and call `start_puzzle('door')` _again_.

I think this could only happen when exercising the new code. Did you observe this while testing it?